src/charactersheet/viewmodels/common/player_card/index.js

```diff
@@ -63,8 +64,16 @@ export class PlayerCardViewModel extends ViewModel {
 
     statusIndicatorClass = pureComputed(() => (
         this.isOnline() ? 'success' : 'failure'
-    ))
+    ));
 
+    popoverHtml() {
+        if (!this.player.isActivePatron) {
+            return '';
+        }
+        const name = this.player.name || 'Player';
+        const tier = capitalize(`${this.player.canonicalPatreonTier} Tier`);
```

Comment:
There's no way for this field to already come from the api in a "display/human readable" format so that the `capitalize` doesnt need to be done?

Reply:
We totally can. Right now I just return the enum because the backend doesnt know the names of the patreon tiers. We could add that.